src/plugins/data-card-tool/data-card-content.ts

```diff
@@ -102,6 +102,15 @@ export const DataCardContentModel = ToolContentModel
     attrById(str: string){
       return self.dataSet.attrFromID(str);
     },
+    isEmptyCase(caseId: string){
+      let attributesWithValues = 0;
+      this.existingAttributes().forEach((attr) => {
+        if (self.dataSet.getValue(caseId, attr) !== "" || undefined){
```

Comment:
The test as written won't work as expected
```suggestion
        const value = self.dataSet.getValue(caseId, attr);
        if (value !== "" && value != null) {
```

Reply:
Ah, I see: https://javascript.info/logical-operators#or-finds-the-first-truthy-value